registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/dto/RestRequestDTO.java

```diff
@@ -0,0 +1,38 @@
+package io.mosip.registration.processor.stages.dto;
+
+import lombok.Data;
+
+import java.util.Map;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.util.MultiValueMap;
+
+@Data
+public class RestRequestDTO {
+
+	@Pattern(regexp = "<\\b(https?|ftp|file)://[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]>", message = "{mosip.rest.request.uri.message}")
```

Comment:
please test invalid pattern and verify if the exception is handled in case of invalid pattern error is thrown